cs-couchbase/src/main/java/io/cloudslang/content/couchbase/entities/couchbase/NodesUri.java

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * (c) Copyright 2017 Hewlett-Packard Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *******************************************************************************/
+package io.cloudslang.content.couchbase.entities.couchbase;
+
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+
+/**
+ * Created by TusaM
+ * 5/9/2017.
+ */
+public enum NodesUri {
+    FAIL_OVER_NODE("FailOverNode", "/controller/failOver");
+
+    private final String key;
+    private final String value;
+
+    NodesUri(String key, String value) {
+        this.key = key;
+        this.value = value;
+    }
+
+    public static String getNodesUri(String input) {
```

Comment:
I'd call the input param `key` and the method `getValue()` -> `NodesUri.getValue(key)`'s responsability is clear to me at first sight